learning_assistant/urls.py

```diff
@@ -19,4 +19,9 @@
         LearningAssistantEnabledView.as_view(),
         name='enabled',
     ),
+    re_path(
+        fr'learning_assistant/v1/course_id/{COURSE_ID_PATTERN}/history/<int:message_count>',
```

Comment:
the message count shouldn't be specified in the path, in my opinion. Happy to be convinced otherwise, but it feels more appropriate, as the message count param is used to filter on the resource being specified.

Reply:
Just realizing query parameters don't need to be in the URL... I'll get rid of that now